src/http_client.rs

```diff
@@ -156,6 +156,15 @@ impl HTTPClient {
                 .or_else(|_| Err("HTTP Request was cancelled".into()))
                 .and_then(|res| {
                     res.and_then(|res| {
+                        if res.status.is_redirection() {
```

Comment:
Should we follow the redirect instead? or see the warn in logs when things break and then update?

Reply:
yeah, seems simplest for now to just return the redirection, and force the consumer of this to realize that route may redirect, and use `post_void` instead to get the headers back and decide whether to follow or not.